requirements.txt

```diff
@@ -1,13 +1,13 @@
-cython==0.29.26
+cython
```

Comment:
Unpinning all the dependencies is a fairly big change. Is it necessary for this commit? If not, I would do in a separate PR.

Reply:
When I added `pyope`, I had conflicting dependency versions. I will try to see how many I can re-pin.

Reply:
Fixed